notebooks/Deploy cohere model.ipynb

```diff
@@ -142,9 +142,6 @@
     "\n",
     "real_time_inference_instance_type = (\n",
     "    \"ml.g5.xlarge\"\n",
-    ")\n",
-    "batch_transform_inference_instance_type = (\n",
```

Comment:
Don't know why this is part of the notebook to begin with

Reply:
Yeah, i don't know either. We for sure don't support it yet.